scripts/Test-Signatures.ps1

```diff
@@ -0,0 +1,297 @@
+[CmdletBinding()]
+param (
+    [String]
+    $Path = "$PSScriptRoot\packages",
+
+    [Switch]
+    $CIBuild
+)
+
+Add-Type -AssemblyName System.IO.Compression.FileSystem

```

Comment:
Is this used to work with archives? There are Compress-Archive and Expand-Archive cmdlets since powershell 5, I'd use that if possible.

Reply:
They don't work if the file extension is not a valid archive.

Reply:
kk